trunk/SUAVE/Analyses/Mission/Segments/Ground/Takeoff.py

```diff
@@ -78,10 +79,24 @@ def __defaults__(self):
         # --------------------------------------------------------------
         #   The Solving Process
         # --------------------------------------------------------------
-    
+
+        # --------------------------------------------------------------
+        #   Initialize - before iteration
+        # --------------------------------------------------------------
         initialize = self.process.initialize
-        initialize.conditions = Methods.Ground.Takeoff.initialize_conditions
+        initialize.conditions              = Methods.Ground.Takeoff.initialize_conditions
         
+        # --------------------------------------------------------------
+        #   Finalize - after iteration
+        # --------------------------------------------------------------
+        finalize = self.process.finalize
+
+        # Post Processing
+        finalize.post_process = Process()
+        finalize.post_process.inertial_position = Methods.Common.Frames.integrate_inertial_horizontal_position
+        finalize.post_process.noise             = Methods.Common.Noise.compute_noise
```

Comment:
I included the addition of takeoff and landing to the range. This allows the position of the aircraft to be recorded so that I can perform noise analysis when the aircraft is accelerating on the ground. Similar to FAA sideline noise etc.

Reply:
Addressed. I reverted the changes.